### DISCUSSION TITLE
Scraping Historic Market Values

### DISCUSSION THREAD

**Discussion by olitsch123** [dcaribou/transfermarkt-datasets#165] [Q&A] [answered]
Hi! I m completely new to coding and try to extract historic market values from https://www.transfermarkt.de/kai-havertz/marktwertverlauf/spieler/309400  Can anybody help?

**Comment by dcaribou** [comment, marked as answer]
Hi @olitsch123,

Historic market values are provided in the `player_valuations` asset. You can download it from Kaggle or data.world (checkout the project README).

Here's how it looks like in the [player value sample analysis ](https://transfermarkt-datasets.fly.dev/analysis:_player_value)in the streamlit app.

<img width="1283" alt="image" src="https://github.com/dcaribou/transfermarkt-datasets/assets/16071835/fa80a260-a7dc-4c78-84cc-7973c10cd31e">

**Comment by Charlotte-br560** [comment]
Sure thing! You're on the right track. Check out Crawlbase for seamless scraping. It's newbie-friendly too!
